docs/units/logarithmic_units.rst

```diff
@@ -137,6 +137,29 @@ appropriate equivalency::
     >>> V.to(u.ABmag, u.spectral_density(5500.*u.AA))  # doctest: +FLOAT_CMP
     <Magnitude [16.99023831, 21.10978201, 22.50053827] mag(AB)>
 
+This is particularly useful for converting magnitude into flux density.
+``V`` is currently in ST magnitudes, which is based on flux densities per
+unit wavelength::
+
+    >>> flam = V.to(u.erg/u.s/u.cm**2/u.AA)
+    >>> flam # doctest: +FLOAT_CMP
+    <Quantity [5.75439937e-16, 1.29473986e-17, 3.59649961e-18] erg / (Angstrom cm2 s)>
+
+To convert ``V`` to flux density per unit frequency (on which AB magnitudes
+are based), we again need the appropriate equivalency::
+
+    >>> fnu = V.to(u.erg/u.s/u.cm**2/u.Hz, u.spectral_density(5500*u.AA))
```

Comment:
Why not define `lam = 5500 * u.AA` before this line and then use it here as well?
```suggestion
    >>> lam = 5500 * u.AA
    >>> fnu = V.to(u.erg/u.s/u.cm**2/u.Hz, u.spectral_density(lam))
```